services/storage/src/simcore_service_storage/modules/celery/errors.py

```diff
@@ -0,0 +1,23 @@
+import base64
+import pickle
+
+
+class TransferrableCeleryError(Exception):
+    def __repr__(self) -> str:
+        exception = decode_celery_transferrable_error(self)
+        return f"{self.__class__.__name__}({exception.__class__.__name__}({exception}))"
+
+    def __str__(self) -> str:
+        return f"{decode_celery_transferrable_error(self)}"
+
+
+def encore_celery_transferrable_error(error: Exception) -> TransferrableCeleryError:
```

Comment:
did you have some tests around these things?

Reply:
I've just extended testing of this mechanism inside the test_async_jobs.py